# int8 class_id
string class_name
float32 score
bool tracked
int32 id

geometry_msgs/Pose2D center
float32 size_x
float32 size_y